lib/createlocaltrack.js

```diff
@@ -31,7 +31,7 @@ function createLocalTrack(kind, options) {
 /**
  * Request a {@link LocalAudioTrack}.
  * @alias module:twilio-video.createLocalAudioTrack
- * @param {CreateLocalTrackOptions} [options] - Options for requesting a {@link LocalAudioTrack}
+ * @param {CreateLocalAudioTrackOptions} [options] - Options for requesting a {@link LocalAudioTrack}
```

Comment:
This doesn't match the ts defs

Reply:
Thanks, tsdefs are now fixed.

Reply:
I think instead of removing `CreateLocalTrackOptions` in tsdefs, we should update this instead to `CreateLocalTrackOptions|CreateLocalAudioTrackOptions `. Because now the tsdef changes are breaking changes.

Reply:
addressed now.